pkgs/applications/science/programming/scyther/cli.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, cmake, glibc, makeWrapper, fetchFromGitHub, flex, bison, source }:
+stdenv.mkDerivation { 
+  name = "scyther-cli-${source.rev}";
+  src = source;
+  preConfigure = ''
+    cd src
+    # Since we're not in a git dir, the normal command this project uses to create this file wouldn't work
+    printf "#define TAGVERSION \"${source.rev}\"\n" >> version.h
+
+    # For some reason it doesn't use the C89 standard which would allow __inline__
+    for file in *.*
+    do
+      substituteInPlace $file --replace "__inline__ " ""
```

Comment:
The issue about this version.h file comes from the fact that the python script for its generation relies on being in a git repo (so it can check the tag/rev). This could be solved by doing a full checkout, but I don't think it's worth the build time dependency of python, considering the script [is dead simple](https://github.com/cascremers/scyther/blob/master/src/describe-version.py#L21-L45).

The issue with these `__inline__` attributes I have no idea how to solve, cmake is [supposed to use the C89 standard](https://github.com/cascremers/scyther/blob/master/src/CMakeLists.txt#L30-L32) but it doesn't for some reason. If somebody could show me how it's done that would be awesome.